instituto-de-investigaciones-juridicas-unam.csl

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="es-MX">
+  <info>
+    <title>Instituto de Investigaciones Jurídicas UNAM (es-MX)</title>
+    <title-short>IIJ-UNAM</title-short>
+    <id>http://www.zotero.org/styles/instituto-de-investigaciones-juridicas-unam</id>
+    <link href="http://www.zotero.org/styles/instituto-de-investigaciones-juridicas-unam" rel="self"/>
+    <link href="https://revistas.juridicas.unam.mx/index.php/derecho-internacional/libraryFiles/downloadPublic/34" rel="documentation"/>
+    <author>
+      <name>Coordinación de Revistas</name>
+      <email>revistas.iij@unam.mx</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <category field="political_science"/>
+    <summary>Instituto de Investigaciones Juridicas de la UNAM</summary>
+    <updated>2025-08-21T21:23:48+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="es">
+    <date form="text">
+      <date-part name="year" suffix="."/>
+    </date>
+    <terms>
+      <term name="chapter" form="short">cap. </term>
```

Comment:
This term already is "cap." as per our es-MX locale and the space is already included in your point-locator macro group. Please delete this line.